js/src/vue/Kanban/Column.vue

```diff
@@ -408,8 +408,8 @@
     .kanban-column {
         margin-right: 16px;
         border-radius: 5px;
+        width: 400px;
         flex-direction: column;
-        flex: 0 0 350px;
```

Comment:
This rule never had an effect since the column element is block displayed, not flex.